WordPress/Classes/Models/Blog.m

```diff
@@ -458,6 +458,8 @@ - (BOOL)supports:(BlogFeature)feature
             return [self isHostedAtWPcom] && [self supportsSiteManagementServices];
         case BlogFeatureNoncePreviews:
             return [self supportsRestApi] && ![self isHostedAtWPcom];
+        case BlogFeatureMediaMetadataEditing:
+            return [self isHostedAtWPcom];
```

Comment:
Thanks, that makes sense. Technically `isAdmin` isn't right, as you can edit metadata as an author or editor – but we don't currently have any way to detect that, so let's stick with admin for now.